src/main/kotlin/com/arabyte/arabyteapi/domain/article/service/ArticleService.kt

```diff
@@ -82,7 +82,7 @@ class ArticleService(
         val article = articleRepository.findById(articleId)
             .orElseThrow { CustomException(CustomError.ARTICLE_NOT_FOUND) }
 
-        val user = userService.getUser(article.userId)
+        val user = userService.getUser(article.user.id)
```

Comment:
```suggestion
        val user = article.user
```

이렇게 하지 않고 db에 한 번 더 거치는 이유가 있나요??

Reply:
아 원래는 article에 userId:Long로 가지고 있던걸 이번에 User테이블이랑 연관관계를 만들면서 연관되는 코드들 수정하는 과정에서 실수한 것 같습니다. 감사합니다.